scripts/datadog_wrapper

```diff
@@ -86,6 +86,7 @@ then
   export DD_JMXFETCH_ENABLED="false"
   export DD_RUNTIME_METRICS_ENABLED="false"
   export DD_REMOTE_CONFIG_ENABLED="false"
+  export DD_APPSEC_ENABLED="false"
```

Comment:
Will this be overridden if the user sets it to `true`? I'm not aware what's our support parity on the AppSec side...

Reply:
This only disables the tracer-side appsec. For serverless appsec, we use DD_SERVERLESS_APPSEC_ENABLED env var as the implementation is in the agent and not the tracer.

Reply:
Thanks for clarifying!